src/cljs_time/extend.cljs

```diff
@@ -11,7 +11,8 @@
   (:import
    [goog.date Date]
    [goog.date DateTime]
-   [goog.date UtcDateTime]))
+   [goog.date UtcDateTime])
+  (:require [cljs-time.core :as ct]))
```

Comment:
I think all (or most) parts of the lib are using the alias `time`
```clojure
    [cljs-time.core :as time]
```
Could you change it for consistency?